about.html

```diff
@@ -0,0 +1,150 @@
+            <ul class="main_nav_link">
+              <li><a href="index.html" class="sub_home_link">Home</a></li>
+              <li><a href="#" class="sub_nav_link">Program</a></li>
+              <li><a href="#" class="sub_nav_link">Sponsor</a></li>
+              <li><a href="#" class="sub_nav_link">Join</a></li>
+            </ul>
+        </div>
+
+        <div class="sub_nav_container">
+          <a href="#" class="startup_logo"><img class="img_logo" src="capstone-image/new logo.png" alt="startup"></a>
+
+          <div class="mobile_menu_container">
+            <ul class="mobile_nav_link">
+              <li><a href="index.html" class="mobile_nav_link">Home</a></li>
+              <li><a href="#" class="mobile_nav_link">Program</a></li>
+              <li><a href="#" class="mobile_nav_link">Join</a></li>
+              <li><a href="#" class="mobile_nav_link">Sponsor</a></li>
+              <li><a href="#" class="mobile_nav_link">News</a></li>
+            </ul>
+
+            <button class="nav_button" type="button">ST Campaign</button>
+          </div>
+        </div>
+
+        <div class="hamburger_menu">
+          <span class="ham_bar"></span>
+          <span class="ham_bar"></span>
+          <span class="ham_bar"></span>
+        </div>
+
+      </nav>
+    </header>
+     <div class="main_section_container">
+        <div class="top_section_container">
+        <p class="main_section_title">"Join Our Innovation!!"</p>
+        <h1 class="theme">MEET-UP TO START UP, 2023 TECH CONFERENCE!</h1>
+      </div>
+
+      <div class="main_section_content">
+        <p class="main_section_text">
+          Regie is designed to produce sales and marketing copy for brands to employ more quickly and yet with more consistency and personalization than doing so manually. The startup adapts the GPT-3 model specifically for businesses to use in email campaigns, online ads, text messages, social media, and even podcast ad copy. Keeping up with the demand for all of that content across so many media becomes a lot easier when AI can take a phrase or two as suggestions and rapidly compose multiple messages for each outlet and style of writing after the startup trained its GPT-3 engine on 20,000 sales sequences. Clients can integrate Regie into their own content management systems or use it a browser extension, and the company claims to track thousands of sales sequences and more than 35 million emails. It’s already used by more than 75 brands, including Crunchbase, Hubspot, and AT&T, though the content is checked by a human curator to avoid any issues of unsupervised AI content generation.</p>
+
+          <p class="contact_us">
+            Please contact us now per Email to get started if you are ready join hundreds of entreprenuers for this edition of "MEET-UP TO START UP"
+          </p>
+          <a href="mailto:info@decagoninstitute.com" class="email_add">info@decagoninstitute.com</a>
+
+      </div>
```

Comment:
- [ ] As stated before kindly make sure to be aligned with design guidelines by making the text aligned center and also choosing the same background color for the heading as stated before.

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/234955246-5e877195-77cc-4c6d-8af7-fbe77fee0a52.png)  | ![image](https://user-images.githubusercontent.com/53402600/234955294-4873af21-36c7-484c-8cb6-c36bc5272432.png) |